pyne/tally.pyx

```diff
@@ -21,6 +24,40 @@ from pyne.utils import QAWarning
 
 warn(__name__ + " is not yet QA compliant.", QAWarning)
 
+cdef vector[int] to_vector_int(value):
+    cdef vector[int] value_proxy
+    cdef int ivalue
+    cdef int value_size
+    cdef int * value_data
+    # value is a ('vector', 'int32', 0)
+    value_size = len(value)
+    if isinstance(value, np.ndarray) and (< np.ndarray > value).descr.type_num == np.NPY_INT64:
+        value_data = <int * > np.PyArray_DATA( < np.ndarray > value)
+    value_proxy = vector[int](< size_t > value_size)
+    for ivalue in range(value_size):
+        value_proxy[ivalue] = <int > value[ivalue]
```

Comment:
This won't work because you don't use `value_data`?

Reply:
yes sorry... test were working so I did not look at it further...
should be fixed in the next commit.

Reply:
I am not sure this implementation will work, 
the problem is I cannot cast `value` in `value_data` I think that's why the former implementation looked like that...

Reply:
I re-implemented the previous implementation (with a slight improvement...)

the problem, is as we don't know the type of `value` we cannot cast it directly into value_data (which is a `double*`)